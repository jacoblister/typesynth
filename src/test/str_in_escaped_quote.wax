(func main (result int)
  (let s str "\"a = 1\" next")
  (print s)
  (return 0)
)
