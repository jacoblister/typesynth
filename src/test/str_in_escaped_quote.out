"a = 1" next
